### PR TITLE
REST-API & docs

### PR DESCRIPTION
**Pull request by ocarriles**
Account REST API dependency included

### CODE REVIEW

File: docs/src/main/asciidoc/api/account-api.adoc [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+NOTE: The above command uses the Account SID and the one below uses the Email Account. Note the administrator%40company.com is used instead of administrator@company.com . This is because using curl on the bash terminal doesn't parse the @ correctlyl. If you were to running on a browser, you can safely use the @ as the web browser will correctly handle it.
+
+....
+curl -X GET  http://administrator%40company.com:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf
+....
+
+The above commands will print an output similar to the one below:
+
+----
+<RestcommResponse>
```
Comment (deruelle) [inline docs/src/main/asciidoc/api/account-api.adoc]: The restcomm response below doesn't seem correct as the account in the sbc project doesn't have any notion of AvailablePhoneNumbers, IncomingPhoneNumbers, etc...

File: docs/src/main/asciidoc/api/account-api.adoc [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+**Change default account password(AuthToken).**
+
+....
+curl -X PUT http://ACae6e420f425248d6a26948c17a9e2acf:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf -d "Password=NewPassword"
+....
+
+NOTE: The above command uses the Account SID and the one below uses the Email Account. Note the administrator%40company.com is used instead of administrator@company.com . This is because using curl on the bash terminal doesn't parse the @ correctlyl. If you were to running on a browser, you can safely use the @ as the web browser will correctly handle it.
+
+....
+curl -X GET  http://administrator%40company.com:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf
```
Comment (deruelle) [inline docs/src/main/asciidoc/api/account-api.adoc]: shall we make restcomm context in the URL be sbc instead of restcomm-sbc ?

Comment (ocarriles) [inline docs/src/main/asciidoc/api/account-api.adoc]: sounds reasonable, let's do it if it is consistent with RestComm naming policy

**Comment by deruelle** [comment]
On #29 commits above, we need to add unit tests
on #34 commits above, we call to a script to actually delegate the Firewall whitelisting or blacklisting (https://github.com/RestComm/sbc/pull/36/commits/bab026b1577aa102cf87cee171151bd8aa9daf1f#diff-fa154f0f4c5f4e8efac48543ba57bb19R20), but no default script is present. We need to add some. Ideally one for AWS and one for regular linux ubuntu server box.

**Pull request merged**